src/drt/src/frRegionQuery.cpp

```diff
@@ -731,12 +731,12 @@ void frRegionQuery::Impl::init()
     }
     cnt++;
     if (VERBOSE > 0) {
-      if (cnt < 100000) {
-        if (cnt % 10000 == 0) {
+      if (cnt < 1000000) {
```

Comment:
could this be a `static constexpr int some_name_ = 100000;` in a header (same for all the rest)? might be cleaner to read and easier to update later if we need to.

Reply:
Since we know how much work we are going to do it is easy to convert this to 10% , 20%, etc which is more scale invariant.

Reply:
Good future improvement.